data/json/recipes/recipe_ammo.json

```
@@ -719,7 +719,7 @@
     "time": "60 m",
     "qualities": [ { "id": "CHEM", "level": 2 } ],
     "book_learn": {
-      "textbook_gaswarfare": { "skill_level": 8, "recipe_name": "" },
+      "textbook_gaswarfare": { "skill_level": 8, "recipe_name": "napalm" },
```

You can also skip specifying it.
```suggestion
      "textbook_gaswarfare": { "skill_level": 8 },
```

---

That would lead to a situation where the number of recipes you can learn from the book is disjointed from the number of entries the game attempts to display. Would that lead to a buffer overread when it tries to look for one additional recipe?

---

If this member is not specified, it defaults to the name of the item that the recipe produces.

---

Ahhhh. Today I learned!